Sources/TuistGenerator/Generator/ProjectGenerator.swift

```diff
@@ -248,12 +259,23 @@ final class ProjectGenerator: ProjectGenerating {
                                                    generatedProject: generatedProject)
     }
 
+    enum XcodeVersionError: Swift.Error {
```

Comment:
The same error is defined [here](https://github.com/tuist/tuist/pull/394/files#diff-4deac93067ff0b8e79a12c11e5327605R45). What about reusing it?

Reply:
It was not used, forgot to delete it, good catch, it's removed now.